packages/auth/amplify_auth_cognito_dart/lib/src/util/authorization_providers.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think that would be a good rename. I was originally thinking this could have the cog user pool provider, too, but separating them prob better. I'm reconsidering some aspects of the design here that might change this, however.